src/screens/Login/components/Runner/index.tsx

```diff
@@ -15,22 +15,21 @@ interface RunnerProps {
   logoutAction: () => any
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   alternativeLoginAction?: () => any
-  refresh: () => void
 }
 
 export default function Runner(props: RunnerProps) {
   const { t } = useTranslation()
   async function handleLogout() {
     await props.logoutAction()
-    window.localStorage.clear()
-    props.refresh()
+    // FIXME: only delete local storage relate to one store, or only delete if logged out from both
+    //window.localStorage.clear()
```

Comment:
we have a bug here, but it's outside of the scope of this refactor: when we log out from one store this is clearing the localStorage but we are still logged in with the other store so this shouldn't happend

maybe we have to only delete the data specific to the store that's logged out only?